code/data_structures/stack/balanced_expression/balanced_expression.java

```diff
@@ -0,0 +1,36 @@
+import java.util.*;
+
+class BalancedExpression {
+	public static void main(String args[]) {
+		System.out.println("Balanced Expression");
+		Scanner in = new Scanner(System.in);
+		String input = in.next();
+		if (is_expression_balanced(input)) {
+			System.out.println("The expression is balanced");
+		}
+		else {
+			System.out.println("The expression is not balanced");
+		}
+
+		static boolean is_expression_balanced(String input) {
```

Comment:
Change to isExpressionBalanced like @Croolis  said.

Reply:
Done..